addons/web_tour/static/src/debug/debug_manager.js

```diff
@@ -62,6 +61,13 @@ export function setupDebugTourItems(env) {
 
 class ToursDialog extends Component {
 
+    /**
+     * @returns {string} The dialog title
+     */
+    getDialogTitle() {
```

Comment:
why not getter method?

Reply:
Should have been indeed. But This is already replaced by putting the title on the Dialog tag in the xml. (I work on the same stuff between two branches). title tags in xml are actually automatically added to the translations files. And since the prop is called title, that works too.

Reply:
Little follow up: Owl components tags attributes are not automatically export for translation. We might work on something later to improve that. t t-value are ignored as well, but we can go around it with `<t t-set="something">t value directly as string</t>`